.github/workflows/build.yml

```diff
@@ -220,6 +220,13 @@ jobs:
           PACK_HELP="${PACK_HELP//$'\n'/'%0A'}"
           PACK_HELP="${PACK_HELP//$'\r'/'%0D'}"
           echo "::set-output name=help::$PACK_HELP"
+      - name: Generate SBOM JSON
+        uses: CycloneDX/gh-gomod-generate-sbom@v0.3.0
+        with:
+          json: true
+          output: bom.cdx.json
```

Comment:
Where would the bom be stored? Should it be attached to the release as an artifact?

Reply:
It is attached to the release as an artifact yes - see https://github.com/CycloneDX/cyclonedx-go/releases/tag/v0.2.1

Reply:
I'm confused... how do they accomplish this magic? Are you sure the release notes are referring to __your__ (the user's) CI, not just cyclonedx-go CI?

Reply:
Ah yes sorry, my bad, it looks like they use go releaser to attach the bom https://github.com/CycloneDX/cyclonedx-gomod/blob/0b09de5978d9120a2b1f03ad376e5e0eeaaf9c65/.goreleaser.yml#L54

Reply:
If you follow the naming convention as our other artifacts `pack-v${{ env.PACK_VERSION }}-*`, it would be picked up by our release action on line 246. Can we make that change?

Reply:
Simple and convenient! Thanks for the tip!